Extensions/3D/JsExtension.js

```diff
@@ -800,6 +800,20 @@ module.exports = {
         .markAsSimple()
         .setHidden()
         .setFunctionName('hasAnimationEnded');
+
+      object
+        .addScopedAction(
+          'SetCrossfadeDuration',
+          _('Set crossfade duration'),
+          _('Set the crossfade duration when switching to a new animation.'),
+          'Set crossfade duration of _PARAM0_ to _PARAM1_',
+          _('Duration'),
+          'res/conditions/3d_box.svg',
+          'res/conditions/3d_box.svg'
```

Comment:
Let's use another icon. This action is not related to a 3D box, this will be confusing.